stellar-contract-env-common/src/env.rs

```diff
@@ -183,6 +186,24 @@ macro_rules! call_macro_with_all_host_functions {
                 {"_", fn compute_hash_sha256(x:Object) -> Object}
                 {"0", fn verify_sig_ed25519(x:Object, k:Object, s:Object) -> RawVal}
             }
+
+            mod account "a" {
+                /// Get the low threshold for the account with ed25519 public
+                /// key a (a is binary). Traps if no such account exists.
```

Comment:
>Traps if no such account exists.

Any trap needs to be avoidable, so I guess we need a function such as `account_does_exist`.

Reply:
Agreed. I realized we need the same for `has_invoking_contract`. I'll get to that when I actually start consuming these functions next week.